phoenix-queryserver/pom.xml

```diff
@@ -147,6 +147,14 @@
       <groupId>commons-logging</groupId>
       <artifactId>commons-logging</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.phoenix</groupId>
+      <artifactId>phoenix-queryserver-loadbalancer</artifactId>
```

Comment:
how do you propose sharing these configurations, which is needed by the load balancer and PQS?